src/tests/end-to-end/simple-study.cpp

```diff
@@ -479,3 +592,292 @@ BOOST_AUTO_TEST_CASE(two_mc_year_two_ts_different_weight)
 }
 
 BOOST_AUTO_TEST_SUITE_END()
+
+void prepareStudy(int nbYears, int nbTS, Study::Ptr &pStudy, Area *&area1,
```

Comment:
ok, seemingly some tests are not finished, so my review on the current file stops here.